common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3048,6 +3048,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_RECURSIVE_CTE" : {
+    "message" : [
+      "Invalid recursive definition found. Recursive queries must contain an UNION or an UNION ALL statement with 2 children. The first child needs to be the anchor term without any recursive references. <error>"
```

Comment:
do we really need this `<error>` parameter? It's always `Unsupported recursive CTE UNION placement.` which seems duplicated with the error message here.